src/hooks/index.js

```diff
@@ -0,0 +1,56 @@
+import { getTodoList } from "../api";
+import { useQuery } from "@tanstack/react-query";
+import { useQueryClient, useMutation } from "@tanstack/react-query";
+import { deleteTodo } from "../api";
+import { updateTodo } from "../api";
```

Comment:
You can import `updateTodo` from above line

Reply:
Done.